### ISSUE TITLE
From background to Blending problem

### ISSUE THREAD

**Issue by AzureRossi** [layerdiffusion/sd-forge-layerdiffuse#56]
First, thanks for the great work but I may have the following problem. 
I downloaded the background image you provided and configured it with the same parameters as follows: 
**old man sitting, high quality
Negative prompt: bad, ugly
Steps: 20, Sampler: DPM++ 2M SDE Karras, CFG scale: 7, Seed: 12345, Size: 896x1152, Model hash: 1fe6c7ec54, Model: juggernautXL_version6Rundiffusion, layerdiffusion_enabled: True, layerdiffusion_method: From Background to Blending, layerdiffusion_weight: 1, layerdiffusion_ending_step: 1, layerdiffusion_fg_image: False, layerdiffusion_bg_image: True, layerdiffusion_blend_image: False, layerdiffusion_resize_mode: Crop and Resize, Version: f0.0.17v1.8.0rc-latest-276-g29be1da7**

However, my inference results are as follows: 
![old_man](https://github.com/layerdiffusion/sd-forge-layerdiffuse/assets/106904993/4e5b98d6-ed80-4b60-b664-e654b6b26ef6)

The result always involves the generation of new objects in certain areas, while the geometric structures between layers are not well aligned. I would like to know if this is a problem with **layer_xl_bg2ble.safetensors** or if it's due to some details I might have overlooked personally.
Thanks.

**Comment by DuoLi1999** [comment]
Yeah, am also meeting these situations, have you fix it? Or this is the normal performance?
